src/api/db/migrate/20170407151300_fix_dod_repotype.rb

```diff
@@ -0,0 +1,8 @@
+class FixDodRepotype < ActiveRecord::Migration[5.0]
+  def up
+    DownloadRepository.where(repotype: "rpmmd").update(repotype: "rpm-md")
```

Comment:
You can use [.update_all](https://apidock.com/rails/ActiveRecord/Relation/update_all), should be faster as it does not instantiate and doesn't validate...